lib/galaxy/managers/users.py

```diff
@@ -699,6 +705,25 @@ def add_serializers(self):
             }
         )
 
+    def serialize_disk_usage(self, user: model.User) -> List[Dict[str, Any]]:
+        rval = user.dictify_usage(self.app.object_store)
+        for usage in rval:
+            quota_source_label = usage["quota_source_label"]
+            usage["quota_percent"] = self.user_manager.quota(user, quota_source_label=quota_source_label)
+            usage["quota"] = self.user_manager.quota(user, total=True, quota_source_label=quota_source_label)
+            usage["quota_bytes"] = self.user_manager.quota_bytes(user, quota_source_label=quota_source_label)
+            usage["nice_total_disk_usage"] = util.nice_size(usage["total_disk_usage"])
```

Comment:
I don't recall why I added these - likely because the UI worked differently at the time or maybe just for consistency with the old API but I'm happy to drop them. I'll do that on a rebase.